playwright-e2e/tests/dsm/kitUploadFlow/saliva-kit-upload-flow.spec.ts

```diff
@@ -0,0 +1,180 @@
+        if (textData.split('\n').length < 28) {
+          testParticipantIndex = count;
+          break;
+        }
+      }
+
+      // Collects all the necessary data for kit upload
+      const participantPage: ParticipantPage = await participantListTable.openParticipantPageAt(testParticipantIndex);
+      shortID = await participantPage.getShortId();
+      const isContactInformationTabVisible = await participantPage.isTabVisible(TabEnum.CONTACT_INFORMATION);
+      kitUploadInfo = new KitUploadInfo(
+        shortID,
+        await participantPage.getFirstName(),
+        await participantPage.getLastName(),
+      );
+
+      // collects data from the contact information tab if the tab is available
+      if (isContactInformationTabVisible) {
+        const contactInformationTab = await participantPage.clickTab<ContactInformationTab>(TabEnum.CONTACT_INFORMATION);
+        kitUploadInfo.street1 = await contactInformationTab.getStreet1();
+        kitUploadInfo.city = await contactInformationTab.getCity();
+        kitUploadInfo.postalCode = await contactInformationTab.getZip();
+        kitUploadInfo.state = await contactInformationTab.getState();
+        kitUploadInfo.country = await contactInformationTab.getCountry();
+      }
+
+      // deactivate all kits for the participant
+      const kitsWithoutLabelPage = await navigation.selectFromSamples<KitsWithoutLabelPage>(SamplesNavEnum.KITS_WITHOUT_LABELS);
+      await kitsWithoutLabelPage.waitForLoad();
+      await kitsWithoutLabelPage.assertPageTitle();
+      await kitsWithoutLabelPage.selectKitType(kitType);
+      await kitsWithoutLabelPage.assertCreateLabelsBtn();
+      await kitsWithoutLabelPage.assertReloadKitListBtn();
+      await kitsWithoutLabelPage.assertTableHeader();
+      await kitsWithoutLabelPage.deactivateAllKitsFor(shortID);
+
+      // Uploads kit
+      const kitUploadPage = await navigation.selectFromSamples<KitUploadPage>(SamplesNavEnum.KIT_UPLOAD);
+      await kitUploadPage.waitForLoad();
+      await kitUploadPage.assertPageTitle();
+      await kitUploadPage.assertDisplayedKitTypes(expectedKitTypes);
+      await kitUploadPage.selectKitType(kitType);
+      await kitUploadPage.assertBrowseBtn();
+      await kitUploadPage.assertUploadKitsBtn();
+      await kitUploadPage.assertInstructionSnapshot();
+      await kitUploadPage.uploadFile(kitType, [kitUploadInfo], study);
+
+      // initial scan
+      const initialScanPage = await navigation.selectFromSamples<InitialScanPage>(SamplesNavEnum.INITIAL_SCAN);
+      await initialScanPage.assertPageTitle();
+      kitLabel = `kit-${crypto.randomUUID().toString().substring(0, 10)}`;
+      await initialScanPage.fillScanPairs([kitLabel, shortID]);
+      await initialScanPage.save();
+
+      // Kits without label for extracting a shipping ID
+      await navigation.selectFromSamples<KitsWithoutLabelPage>(SamplesNavEnum.KITS_WITHOUT_LABELS);
+      await kitsWithoutLabelPage.waitForLoad();
```

Comment:
After `waitForLoad()`, is there a page title where you can do `kitsWithoutLabelPage.assertPageTitle()`?

Reply:
You can find that assertion on line 134. The title is there always